### ISSUE TITLE
Make it work for parameterized tests

### ISSUE THREAD

**Issue by yotov** [microsoft/playwright-vscode#54]
I'm using a for loop to iterate over an array with test data to run the same test multiple times and it's detected properly by the test runner extension. But starting the test run it exponentially - for every test that is detected in the loop it runs all the test in the loop

**Comment by Daemos87** [comment]
Same here, 

Running the following tests with the test runner extensione execute 16 tests 

``` typescript

for (let years of [7, 8, 9, 10]) {
    test(`Product not visible for ${years} duration`, async ({ risultati }) => {
        await risultati.checkDurata(years);
        await expect(
            risultati.page.locator(`a.ZZ_button[data-prodotto-id="${risultati.product_ids.permicro}"]:visible`)
        ).toBeHidden({ timeout: 5000 });
    });
}
```

Running the same tests with the command line execute the expected 4 tests.

**Comment by pavelfeldman** [comment]
Should be fixed 0.1.0.
